content/docs/command-reference/stage/add.md

```diff
@@ -37,82 +37,72 @@ are ignored by `dvc stage add`.
+[defining pipeline stages]:
+  /doc/user-guide/pipelines/defining-pipelines#dvcyaml-metafiles
+[dependency graph]:
+  /doc/user-guide/pipelines/defining-pipelines#directed-acyclic-graph-dag
 
 ### Dependencies and outputs
 
-By specifying lists of <abbr>dependencies</abbr> (`-d` option) and/or
-<abbr>outputs</abbr> (`-o` and `-O` options) for each stage, we can create a
-[dependency graph] that connects them, i.e. the output of a stage becomes the
-input of another, and so on (see `dvc dag`). This graph can be restored by DVC
-later to modify or [reproduce](/doc/command-reference/repro) the full pipeline.
-For example:
-
-```cli
-$ dvc stage add -n printer -d write.sh -o pages ./write.sh
-$ dvc stage add -n scanner -d read.sh -d pages -o signed.pdf ./read.sh pages
-```
-
 Stage dependencies can be any file or directory, either untracked, or more
 commonly tracked by DVC or Git. Outputs will be tracked and <abbr>cached</abbr>
 by DVC when the stage is run. Every output version will be cached when the stage
-is reproduced (see also `dvc gc`).
-
-Relevant notes:
+is reproduced (see also `dvc gc`). Relevant notes:
 
 - Typically, scripts to run (or possibly a directory containing the source code)
   are included among the specified `-d` dependencies. This ensures that when the
   source code changes, DVC knows that the stage needs to be reproduced. (You can
   chose whether to do this.)
 
-- `dvc stage add` checks the dependency graph integrity before creating a new
+- `dvc stage add` checks the [dependency graph] integrity before creating a new
   stage. For example: two stage cannot specify the same output or overlapping
   output paths, there should be no cycles, etc.
 
 - DVC does not feed dependency files to the command being run. The program will
-  have to read by itself the files specified with `-d`.
+  have to read the files itself.
 
 - Entire directories produced by the stage can be tracked as outputs by DVC,
-  which generates a single `.dir` entry in the cache (refer to
-  [Structure of cache directory](/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory)
-  for more info.)
+  which generates a single `.dir` entry in the cache (refer to [Structure of
+  cache directory] for more info.)
 
-- [external dependencies](/doc/user-guide/data-management/importing-external-data)
-  and [external outputs](/doc/user-guide/data-management/managing-external-data)
-  (outside of the <abbr>workspace</abbr>) are also supported (except metrics and
-  plots).
+- [external dependencies] and [external outputs] (outside of the
+  <abbr>workspace</abbr>) are also supported (except metrics and plots).
 
-- Outputs are deleted from the workspace before executing the command (including
-  at `dvc repro`) if their paths are found as existing files/directories (unless
-  `--outs-persist` is used). This also means that the stage command needs to
-  recreate any directory structures defined as outputs every time its executed
-  by DVC.
```

Comment:
@jorgeorpinel Do you mind explaining why you dropped this note?

Reply:
Yeah I was just making it shorter I guess. Idk this was a long time ago. I've reinstated the first part so it's explicit again. PTAL